.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/foundry-rs/forge-std
 [submodule "lib/flexible-voting"]
 	path = lib/flexible-voting
-	url = https://github.com/ScopeLift/flexible-voting
+	url = https://github.com/scopelift/flexible-voting
```

Comment:
Not sure why git changed the casing here 🤷‍♂️

Reply:
`git submodule status` now returns

```
 e5de2efd1368387b840931f19f3c184c85842761 lib/flexible-voting (v1.1.0)
 fc560fa34fa12a335a50c35d92e55a6628ca467c lib/forge-std (v1.5.3)
 1b3adf677e7e383cc684b5d5bd441da86bf4bf1c lib/solmate (v6-186-g1b3adf6)
```

Reply:
I have seen this after running a forge install with a path that has a different casing

Reply:
I ran forge install and am getting 
![Screenshot from 2023-07-27 19-51-30](https://github.com/gitcoinco/Alpha-Governor-Upgrade/assets/12705146/d9557470-df13-4e00-8119-faa30c4a1d64)

Reply:
The hash is right but tag is wrong for some reason maybe I have a different foundry version